client/src/components/jobDetails/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import './index.css';
+
+function jobDetails() {
```

Comment:
react component should named as `PascalCase`, So rename component to `JobDetails`